### PR TITLE
Adding edge events and new edge package

### PR DESCRIPTION
**Pull request by amadt**
## Description

Adding in a new package for edge events. Adding in a couple mobile specific edge SDK events.

## How Has This Been Tested?

No tests needed. Just new schemas.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: packages/edge/schemas/analyticsHit.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "http://griffon.adobe.com/schemas/aep-mobile/analyticsHit",
```
Comment (dgrossen) [inline packages/edge/schemas/analyticsHit.json:2]: should be `schemas/edge/analyticsHit`?

File: packages/edge/schemas/analyticsMapping.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "http://griffon.adobe.com/schemas/aep-mobile/analyticsMapping",
```
Comment (dgrossen) [inline packages/edge/schemas/analyticsMapping.json:2]: should be `schemas/edge/analyticsMapping`?

File: packages/edge/schemas/edgeEvent.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+  "$id": "http://griffon.adobe.com/schemas/aep-mobile/edgeEvent",
```
Comment (dgrossen) [inline packages/edge/schemas/edgeEvent.json:2]: should be `schemas/edge/edgeEvent`?

File: packages/edge/schemas/edgeHitReceived.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "http://griffon.adobe.com/schemas/aep-mobile/edgeHitReceived",
```
Comment (dgrossen) [inline packages/edge/schemas/edgeHitReceived.json:2]: should be `schemas/edge/edgeHitReceived`

File: packages/aep-mobile/src/edgeRequest.js [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+ * Can override defaults and provide additional data by providing a flat object
+ * of paths and values.
+ *
+ * @function
+ * @param {...Function} input Overrides
+ * @returns {object}
+ */
+const mock = (input) => kit.expandWithPaths(path, {
+  datasetId: 'abcdefg',
+  xdm: { 'eventType': 'commerce.purchases' },
```
Comment (dgrossen) [inline packages/aep-mobile/src/edgeRequest.js:239]: linter failing here?

**Comment by amadt** [comment]
Think I actually had some of those changes locally 😄 . Let me push again.

**Review by dgrossen — APPROVED** [review]

**Pull request merged**